components/Workshop/WorkshopForm.tsx

```diff
@@ -78,6 +78,13 @@ export default function WorkshopForm({ user }: {user: User}) {
         try {
           const createdWorkshop = await clientData.createWorkshop(newWorkshop);
           if (createdWorkshop.id) {
+            toast({
+              title: "Workshop created successfully",
+              description: "You can now schedule the workshop",
```

Comment:
Maybe paraphrase this a litle bit to something like "You can now schedule your workshop's availability" or "Add date and time slots to make your workshop live"?

You can try both and see whether they are too long, we probably don't want them breaking on multiple lines. 
Happy for you to make a decision :+1:

Reply:
Done